main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"os"
 
 	//https://linguinecode.com/post/how-to-import-local-files-packages-in-golang
+
```

Comment:
Please don't add blank lines when they're not needed. This can cause merge conflicts since it's in main.